src/Select.js

```diff
@@ -118,10 +118,10 @@ var Select = React.createClass({
 				return;
 			}
 			var menuElem = React.findDOMNode(this.refs.selectMenuContainer);
-			var controlElem = React.findDOMNode(this.refs.control);
+			var wrapperElem = React.findDOMNode(this.refs.control);
```

Comment:
the same case as in `standalone.js`:
`this.refs.wrapper` instead of `this.refs.control`